pkg/auth.go

```diff
@@ -9,6 +9,12 @@ import (
 	"sync"
 	"time"
 
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+
+	types2 "github.com/nuts-foundation/nuts-auth/pkg/types"
```

Comment:
why is this called types2 when there's no 'types' package import?

Also, your formatting seems to be off, I don't remember the Go formatter putting newlines between imports?

Reply:
types2 is fixed in a later commit.
Those imports are weird, its the behaviour of goimports. I'm lookin into that.